src/schema/v2/me/bidder_position_messages.ts

```diff
@@ -1,59 +1,59 @@
 // the description_md must be a function to delay interpolation of string literal
 
 interface BiddingMessage {
-  id:
+  status:
     | "OUTBID"
     | "RESERVE_NOT_MET"
     | "SALE_CLOSED"
     | "LIVE_BIDDING_STARTED"
     | "BIDDER_NOT_QUALIFIED"
     | "ERROR"
-  gravity_key: string
+  message: string
   header: string
-  description_md: (opts: any) => string
+  getDescription: (props?: { liveAuctionUrl: string }) => string
 }
 
-export const BiddingMessages: BiddingMessage[] = [
+export const bidderPositionMessages: BiddingMessage[] = [
   {
-    id: "OUTBID",
-    gravity_key: "Please enter a bid higher than",
-    header: "Your bid wasn’t high enough",
-    description_md: () =>
-      `Another bidder placed a higher max bid\nor the same max bid before you did.`,
+    status: "OUTBID",
+    message: "Please enter a bid higher than",
+    header: "Your bid wasn't high enough",
+    getDescription: () =>
+      `Another bidder placed a higher max bid or the same max bid before you did.`,
   },
   {
-    id: "RESERVE_NOT_MET",
-    gravity_key: "Please enter a bid higher than",
-    header: "Your bid wasn’t high enough",
-    description_md: () =>
+    status: "RESERVE_NOT_MET",
+    message: "Please enter a bid higher than",
+    header: "Your bid wasn't high enough",
+    getDescription: () =>
       `Your bid is below the reserve price. Please select a higher bid.`,
   },
   {
-    id: "SALE_CLOSED",
-    gravity_key: "Sale Closed to Bids",
+    status: "SALE_CLOSED",
+    message: "Sale Closed to Bids",
     header: "Lot closed",
-    description_md: () =>
-      "Sorry, your bid wasn’t received\nbefore the lot closed.",
+    getDescription: () =>
+      "Sorry, your bid wasn't received before the lot closed.",
```

Comment:
Oh — I guess not. Yeah; just a quick run through to use correct punctuation would be nice